preview/mail-processing.html

```diff
@@ -0,0 +1,320 @@
+        <link rel="stylesheet" type="text/css" href="mail-processing_files/ie9.css">
+        <link rel="shortcut icon" href="http://127.0.0.1:4000/images/james-logo.png">
+    </head>
+<body data-gr-c-s-loaded="true">
+    <!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements. See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership. The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied. See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+<link href="mail-processing_files/lightbox_002.html" rel="stylesheet">
+<link href="mail-processing_files/lity_002.html" rel="stylesheet">
+<div id="wrapper">
+  <div class="apache_ref">
+    <a href="https://www.apache.org/" alt="apache foundation link"><img src="mail-processing_files/asf_logo.svg" title="apache foundation logo"></a>
+  </div>
+  <div class="apache_ref_mobile">
+    <a href="https://www.apache.org/" alt="apache foundation link">The Apache Software Foundation</a>
+  </div>
+  <div class="apache_ref_left">
+    <a href="https://www.apache.org/events/current-event.html" alt="apache foundation event"><img src="mail-processing_files/current-event-234x60.png" title="apache foundation event logo"></a>
+  </div>
+  <div class="apache_ref_left_mobile">
+    <a href="https://www.apache.org/events/current-event.html" alt="apache foundation event"><img src="mail-processing_files/current-event-234x60.png" title="apache foundation event logo"></a>
+  </div>
+
+  <!-- Header -->
+    <header id="header" class="alt">
+      <div class="logo"><a href="http://127.0.0.1:4000/index.html" alt="Apache James"><img src="mail-processing_files/james.svg" alt="james logo"></a></div>
+      <h1 class="hidden">James Enterprise Mail Server</h1>
+      <h2>Emails at the heart of your business logic</h2>
+    </header>
+
+  <!-- Main -->
+    <div id="main">
+
+      <!-- Introduction -->
+        <section id="intro" class="main special">
+          <div class="">
+            <div class="content align-left">
+              <header class="major">
+                <h1>How to customize mail processing...</h1>
+              </header>
+                <header class="major">
+                    <h2><b>Mail processing component overview</b></h2>
+                </header>
+              <p class="align-left">At the heart of James lies the Mailet container, which allows mail processing. This is
```

Comment:
I think `lies` -> `lies on`

Reply:
I don't think, it should be the case without the _At_ word at the beginning of the sentence.